README.md

```diff
@@ -1,739 +1,83 @@
+![GitHub contributors](https://img.shields.io/github/contributors/Breakthrough-Energy/REISE.jl?logo=GitHub)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/Breakthrough-Energy/REISE.jl?logo=GitHub)
+![GitHub last commit (branch)](https://img.shields.io/github/last-commit/Breakthrough-Energy/REISE.jl/develop?logo=GitHub)
+![GitHub pull requests](https://img.shields.io/github/issues-pr/Breakthrough-Energy/REISE.jl?logo=GitHub)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![Code of Conduct](https://img.shields.io/badge/code%20of-conduct-ff69b4.svg?style=flat)](https://breakthrough-energy.github.io/docs/communication/code_of_conduct.html)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4538590.svg)](https://doi.org/10.5281/zenodo.4538590)
 
-```
-  -o OUTPUT_DIR, --output-dir OUTPUT_DIR
-                        The directory to store the extracted data. This is
-			optional and defaults to the execute directory.
-			This flag is only used if the extract-data flag is set.
-  -m MATLAB_DIR, --matlab-dir MATLAB_DIR
-                        The directory to store the modified case.mat used by
-			the engine. This is optional and defaults to the execute
-			directory. This flag is only used if the extract-data
-			flag is set.
-  -k, --keep-matlab     The result.mat files found in the execute directory will
-      			be kept instead of deleted after extraction. This flag
-			is only used if the extract-data flag is set.
-```
 
-### Compatibility with PowerSimData
+# REISE.jl
+**REISE.jl** is the Renewable Energy Simulation Engine developed by [Breakthrough
+Energy Sciences](https://science.breakthroughenergy.org/) (BES) to solve DCOPF
+problems. It is written and Julia and can be used with the BES software ecosystem (see
+[PowerSimData]) or in a standalone
+mode (see [Zenodo](https://doi.org/10.5281/zenodo.4538590) for sample data).
 
-Within the Python code in this repo, there is some code to maintain
-compatibility with the `PowerSimData` framework.
 
-Both `call.py` and `extract_data.py` can be called using a positional
-argument that corresponds to a scenario id as generated by the
-`PowerSimData` framework. Using this invocation assumes the presence
-of the `PowerSimData` infrastructure including both a Scenario List
-Manager and Execute List Manager. This option is not intended for manual
-simulation runs.
+## Main Features
+Here are a few things that **REISE.jl** can do:
+* Formulate Production Cost Model (PCM) into an optimization problem that can be solved
+  by professional solvers compatible with [JuMP]
+* Decompose a model that couldn't be solved all at once in memory (due to high spatial
+  or temporal resolution or both) into a series of shorter-timeframe intervals that are
+  automatically run sequentially.
+* Handle storage and demand flexibility
```

Comment:
In the first bullet point, should there be an "a" before "Production Cost Model"?

In the second bullet point, should "couldn't" be "can't"?

In the third bullet point, should we elaborate a little? Perhaps something like "Model operational decisions by energy storage and price-responsive flexible demand alongside those made by thermal and renewable generators"?

Reply:
Done